### PR TITLE
Add cards function.

### PR DESCRIPTION
**Pull request by Dwitsg**
> Changelog

Angular:
- Now the choosewonder redirect to gameboard when all player press in ready.
- Now we can see the cards in gameboard, in correct order and integrated with the backend

Core
- The core now create a game(GameBoard) and this have its own file for routes (GameRoute) and have the own bean (GameService)
- Has been added more models for new features.

 -PD: Please look the models content before creating a new one
 -PD: We have some bug with models (Repeat code)

### CODE REVIEW

File: ui/src/main/javascript/app/gameboard/gameboard.component.js [hunk 1]
```diff
@@ -4,12 +4,13 @@ angular.
     module('sevenWonder.gameboard').
     component('gameboard', {
         templateUrl: 'gameboard/gameboard.tpl.html',
-        controller: ['GameBoard', 'Auth',
-            function GameBoardController(GameBoard, Auth) {
```
Comment (jjohxx) [inline ui/src/main/javascript/app/gameboard/gameboard.component.js:8]: Why did you remove Auth dependecy? 
we need for get the current user.

File: core-service/src/main/java/org/fundacionjala/sevenwonders/beans/GameService.java [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
+import org.fundacionjala.sevenwonders.core.rest.CardModel;
+import org.fundacionjala.sevenwonders.core.rest.DeckModel;
+import org.fundacionjala.sevenwonders.core.rest.PlayerModel;
+import org.fundacionjala.sevenwonders.core.rest.PrincipalGameModel;
 import org.springframework.stereotype.Component;
 
-import java.util.Collection;
-import java.util.Map;
-import java.util.TreeMap;
+import java.util.*;
```
Comment (jjohxx) [inline core-service/src/main/java/org/fundacionjala/sevenwonders/beans/GameService.java:15]: I think that you shouldn't to  import all packages but only the necessary.

**Review by jjohxx — DISMISSED** [review]

File: ui/src/main/javascript/app/gameboard/gameboard.component.js [hunk 3]
```diff
@@ -18,6 +19,31 @@ angular.
                     }, this);
                 });
 
+                this.calculatePosition = function (cards){
+                    var maxCards = 7;
+                    var cardsFinal = [];
+                    var isPair = cards.length % 2 == 0 
+                    var adjustment = isPair ? cards.length + 1 : cards.length;
+                    var pos = Math.abs(Math.floor(-(maxCards/adjustment)));
```
Comment (ziur) [inline ui/src/main/javascript/app/gameboard/gameboard.component.js]: Why is there a negative symbol, I think you won't need the abs function if you remove the negative

**Review by ziur — APPROVED** [review]

**Review by ziur — APPROVED** [review]
